templates/data/postgresql.html

```diff
@@ -1,481 +1,561 @@
+          Use our Charmed PostgreSQL for better control over your TCO. Budgeting and financial planning are straightforward and predictable.
+        </p>
       </div>
     </div>
-  </div>
-</section>
+  </section>
 
-<section class="p-section">
-  <div class="row--50-50">
-    <hr class="p-rule"/>
-    <div class="col p-section--shallow">
-      <h2>What can you achieve<br aria-hidden="true"/> with PostgreSQL?</h2>
-    </div>
-    <div class="col">
-      <p>PostgreSQL is one of the most popular databases. Its popularity is rooted in its sound architectural foundations that make it a good fit for several use cases:</p>
+  <section class="p-section">
+    <hr class="is-fixed-width" />
+    <div class="row p-section--shallow">
+      <div class="col-6 col-medium-3">
+        <h2>Charmed PostgreSQL services</h2>
+      </div>
+      <div class="col-6 col-medium-3">
+        <p>
+          You can self-manage our Charmed PostgreSQL solution or you can opt for a fully managed setup where our Canonical experts will deploy and operate your PostgreSQL fleet on top of your on-premise infrastructure or using your cloud tenants.
+        </p>
+        <hr class="is-muted" />
+        <p>
+          <a href="/contact-us" aria-controls="data-relational-dbs-modal">Get in touch to learn more &rsaquo;&nbsp;</a>
+        </p>
+      </div>
     </div>
-  </div>
-  <div class="row">
-    <div class="col-start-large-4 col-9 col-medium-6">
-      <hr class="p-rule"/>
-      <div class="row">
-        <div class="col-3 col-medium-2">
-          <h3 class="p-heading--5">Web and mobile applications</h3>
-        </div>
-        <div class="col-3 col-medium-2">
-          <p>PostgreSQL is a solid choice for e-commerce and content sharing platforms.</p>
-        </div>
-        <div class="col-3 col-medium-2">
-          {{ 
-            image (
-            url="https://assets.ubuntu.com/v1/2582fc11-Web and mobile applications@3x.png",
-            alt="",
-            width="901",
-            height="600",
-            hi_def=True,
-            loading="lazy"
-            ) | safe
-          }}
+    <div class="row--50-50">
+      <div class="col">
+        <hr class="p-rule--highlight" />
+        <div class="p-card--overlay u-no-padding--top u-no-margin--bottom">
+          <div class="row">
+            <div class="col-4 col-medium-2 col-small-3">
+              <h3 class="u-align-text--left u-no-margin--bottom">Managed</h3>
+            </div>
+            <div class="col-2 col-medium-1 col-small-1">
+              <h3 class="u-align-text--right u-no-margin--bottom">From $3099</h3>
+              <p class="u-align-text--right u-text--muted">Per node, per year</p>
+            </div>
+          </div>
+          <p>
+            Get there faster. Let us deploy and operate your PostgreSQL cluster, in your cloud tenancy or in your data centre.
+          </p>
+          <hr class="is-muted" />
```

Comment:
```suggestion
          <hr class="is-muted p-rule" />
```
There is some extra top padding for the first list item. Adding the `p-rule` should make the paddings for all list items look more consistent.